k8sClient.py

```diff
@@ -76,11 +77,16 @@ def ensure_oc_binary(self) -> None:
     def wait_for_mcp(self, mcp_name: str, resource: str = "resource"):
         time.sleep(60)
         iteration = 1
+        max_tries = 4
         get_status_cmd = "get mcp sriov -o jsonpath='{.status.conditions[?(@.type==\"Updated\")].status}'"
         while self.oc(get_status_cmd).out != "True":
+            if iteration >= max_tries:
+                logger.error(f"mcp {mcp_name} failed to update after {max_tries}, quitting ...")
+                sys.exit(-1)
             start = time.time()
             logger.info(self.oc(f"wait mcp {mcp_name} --for condition=updated --timeout=50m"))
             minutes, seconds = divmod(int(time.time() - start), 60)
-            logger.info(f"It took {minutes}m {seconds}s to for {resource} to be applied (attempt: {iteration})")
             iteration = iteration + 1
             time.sleep(60)
+        
+        logger.info(f"It took {minutes}m {seconds}s for {resource} (attempt: {iteration})")
```

Comment:
(attempt: {iteration}) => (Total Attempts: {iteration})

Reply:
I prefer the more concise `(attempt: {iteration})`